core-kotlin-modules/core-kotlin-collections-4/src/test/kotlin/com/baeldung/maxvalue/MaxValueArrayTest.kt

```diff
@@ -0,0 +1,95 @@
+package com.baeldung.maxvalue
+
+
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatExceptionOfType
+import org.junit.jupiter.api.Test
+
+class MaxValueArrayTest {
+
+    @Test
+    fun givenNonEmptyArray_whenGetMaxOrNull_thenMaxValueIsReturned() {
+
+        val array = listOf(5, 6, 7, 8, 9, 4, 3, 2, 1)
+        val max = array.maxOrNull()
+
+        assertThat(max).isEqualTo(9)
+    }
+
+    @Test
+    fun givenNonEmptyArray_whenGetMaxBy_thenMaxValueIsReturned() {
+
+        val array = listOf(
+            Person("Alan", 34, Height(5, 10)),
+            Person("Jason", 45, Height(5, 11)),
+            Person("Joe", 28, Height(5, 9))
+        )
+        val max = array.maxByOrNull { p -> p.age }
```

Comment:
Insert a blank line here (put blank line between the given/when and when/then part of a test)